thunder/core/jit_ext.py

```diff
@@ -680,6 +681,12 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
         unwrapped_custom_forward_args,
     )
     trace_of_fwd.args = unwrapped_custom_forward_args
+    unpack_bsyms = [
+        prims.unpack_trivial.bind(a, name=a.name, output=a)
+        for a in filter(lambda a: isinstance(a, Proxy), trace_of_fwd.args)
+    ]
+    trace_of_fwd.bound_symbols = unpack_bsyms + trace_of_fwd.bound_symbols
+    trace_of_fwd = dce(trace_of_fwd)
```

Comment:
Do you hit errors without the DCE? It'd be cool to keep the DCE until after the initial trace.

Reply:
let me narrow down the scope of this pr to just correctly propagate exceptions

Reply:
Thank you.